pkgs/development/compilers/elm/packages/node/node-packages.json

```diff
@@ -13,7 +13,6 @@
   "create-elm-app",
   "elm-optimize-level-2",
   "elm-pages",
-  "elm-review",
```

Comment:
Use `./pkgs/development/node-packages/remove-attr.py elm-review` to also remove it from `node-packages.nix`.
And you should add an entry to `pkgs/development/node-packages/aliases.nix`.

Reply:
I ran this, and it doesn't seem to do anything. No output, no error, no changes.

Reply:
We want `nodePackages.elm-review` to remain an alias after we remove it.

Reply:
> I ran this, and it doesn't seem to do anything.

Maybe the entry still needs to exist in `node-packages.json` for the script to work? I'm sure the Python script isn't too hard to read.